examples/rds/aurora-postgres/main.tf

```diff
@@ -0,0 +1,62 @@
+resource "humanitec_application" "app" {
+  id   = var.name
+  name = var.name
+}
+
+module "rds" {
+  source = "../../../humanitec-resource-defs/rds/aurora"
+
+  prefix                 = "${var.name}-"
+  resource_packs_aws_rev = var.resource_packs_aws_rev
+  resource_packs_aws_url = var.resource_packs_aws_url
+
+  access_key = var.access_key
+  secret_key = var.secret_key
+  region     = var.region
+
+  name            = "${var.name}-database"
+  database_name   = "my_database"
+  master_username = "username"
+  master_password = "password"
+
+  vpc                    = var.vpc_id
+  subnets                = var.subnet_ids
+  db_subnet_group_name   = "${var.name}-subnet-group"
+  create_db_subnet_group = true
+  security_group_rules = {
```

Comment:
Is this still needed if we create the group below?

Reply:
RDS in Aurora implementation uses its own SG, not the VPC default one. I removed the group below and make this more restrict